crates/forge_analyzer/src/definitions.rs

```diff
@@ -2617,20 +3037,36 @@ impl Visit for ExportCollector<'_> {
                                     DefRes::Class(()),
                                     ident.as_ref().map(Ident::to_id),
                                 ),
+                                Expr::Ident(ident) => {
+                                    self.add_default(DefRes::Undefined, None);
+                                    // adding the default export, so we can resolve it during the lowering
+                                    self.res_table.exported_names.insert(
+                                        (ident.sym.clone(), self.curr_mod),
+                                        self.default.unwrap(),
```

Comment:
Could you change the ExportCollector::set_default method to return the DefId that was set to self.default at the end of that function?; That way you can do let default_id = self.add_default(DefRes::Undefined, None); and replace self.default.unwrap() with default_id.